bins/revme/src/cmd/evmrunner.rs

```diff
@@ -125,3 +123,36 @@ impl Cmd {
         Ok(())
     }
 }
+
+fn run_benchmark(bytecode_str: Cow<str>) {
+    let evm = Evm::builder()
+        .with_db(BenchmarkDB::new_bytecode(Bytecode::new()))
+        .modify_tx_env(|tx| {
+            tx.caller = address!("1000000000000000000000000000000000000000");
+            tx.transact_to = TxKind::Call(address!("0000000000000000000000000000000000000000"));
+        })
+        .build();
+
+    let host = DummyHost::new(*evm.context.evm.env.clone());
+    let instruction_table = make_instruction_table::<DummyHost, ShanghaiSpec>();
```

Comment:
Having Spec in cli would be best, but for now, putting the latest spec is what would I expect, or at least CancunSpec

Reply:
You are right! Updated to the latest spec